pages/_app.js

```diff
@@ -1,17 +1,33 @@
 import GlobalStyle from "../styles";
 import initialFlashCards from "../lib.data.json";
-import Header from "@/components/Header/Header";
+import useLocalStorageState from "use-local-storage-state";
+import { nanoid } from "nanoid";
+
+import Layout from "@/components/Layout/Layout";
 
 export default function App({ Component, pageProps }) {
+  const [cards, setCards] = useLocalStorageState("cards", { defaultValue: [] });
   const initialData = initialFlashCards;
```

Comment:
```suggestion
 const [cards, setCards] = useLocalStorageState("cards", {
    defaultValue: initialFlashCards,
  });
```
YOU DON'T HAVE TO DO THIS just if you wanted to :)

Reply:
if you want to implement this, you will need to delete all "initialData" props, otherwise you get an error :)